test/cmdlineTests/standard_optimizer_user_yul/input.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity >=0.0;
```

Comment:
The filename/dirname suggests that this uses standard-json

Reply:
It does? How so? The `standard` part?